### ISSUE TITLE
Chat participant names have insufficient color contrast

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#726]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 1.4.3 (AA) Contrast (minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum).

Good contrast between text and background is important for readability, especially for people with impaired vision or while browsing in bright environments.

The participant names in both the main chat view and the expandable participant list have seemingly randomly assigned colors. These colors often lack sufficient color contrast.
![Chat participant list using low-contrast colors.](https://github.com/user-attachments/assets/b17d856c-7ec0-45f2-9094-68fd50a7d7d5)

**To Reproduce**
Open the chat participant list and look at the contrast between text and background.

**Expected behavior / suggestion**
Limit the pool of possible participant name colors so that the color contrast against the background is at least 4.5:1.

For large text (24px, or 19px and bold), at least 3:1 applies.

There are many tools available for checking contrast levels, for example the one at [](https://contrast.report).

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Google Chrome Version 137.0.7151.120 (Official Build) (arm64)

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).
